### DISCUSSION TITLE
Beltrami-flow IC and BC

### DISCUSSION THREAD

**Discussion by tcsmaster** [lululxvi/deepxde#1029] [General]
Can someone help me understand why, in the Beltrami-flow demo, we assign the same function for initial and boundary conditions?
https://github.com/lululxvi/deepxde/blob/master/examples/pinn_forward/Beltrami_flow.py

**Comment by forxltk** [comment]
Because those function are the reference solutions of `u/v/w/p` in the whole domain.

**Reply by tcsmaster** [reply]
But the components are time-depentent, shouldn't the initial condition be the value at t=0? Similar with the boundary conditions?

**Reply by lululxvi** [reply]
The code exactly does this.
